common/src/main/java/org/corfudb/common/metrics/micrometer/README.md

```diff
@@ -20,6 +20,10 @@
 *   **logreplication.opaque.count\_per\_message**: Number of opaque entries per message (rate, mean, max).
 *   **logreplication.opaque.count\_total**: Number of overall opaque entries (rate, mean, max).
 *   **logreplication.opaque.count\_valid**: Number of valid opaque entries (rate, mean, max).
+*   **logreplication.subscribe.trim.count**: Number of times a Trimmed Exception was thrown from the MVO layer when subscribing to LogReplication listener.
+*   **logreplication.subscribe.conflict.count**: Number of times a Transaction Aborted Exception was thrown due to conflicting updates when subscribing to LogReplication listener.
+*   **logreplication.subscribe.duration**: Time taken to subscribe the LogReplication listener.
+*   **logreplication.client.fullsync.duration**: Time taken by the client subscribing to LogReplication listener to perform a full sync on its tables.
```

Comment:
Do we need to have the number of tables involved in the full sync? Assuming that the time taken would be dependent on the table count.

Reply:
Number of tables is just the count of tables of interest. I don't think it should be considered as metrics and viewed from observability perspective.